--- com.microsoft.Edge.desktop.orig
+++ com.microsoft.Edge.desktop
@@ -105,7 +105,7 @@
 Comment[zh_CN]=访问互联网
 Comment[zh_HK]=連線到網際網路
 Comment[zh_TW]=連線到網際網路
-Exec=/usr/bin/microsoft-edge-stable %U
+Exec=/usr/bin/microsoft-edge-stable --ozone-platform=x11 %U
 StartupNotify=true
 Terminal=false
 Icon=microsoft-edge
@@ -172,7 +172,7 @@
 Name[vi]=Cửa sổ Mới
 Name[zh_CN]=新建窗口
 Name[zh_TW]=開新視窗
-Exec=/usr/bin/microsoft-edge-stable
+Exec=/usr/bin/microsoft-edge-stable --ozone-platform=x11
 
 [Desktop Action new-private-window]
 Name=New InPrivate Window
@@ -217,4 +217,4 @@
 Name[vi]=Cửa Sổ InPrivate Mới
 Name[zh_CN]=新建 InPrivate 窗口
 Name[zh_TW]=新 InPrivate 視窗
-Exec=/usr/bin/microsoft-edge-stable --inprivate
+Exec=/usr/bin/microsoft-edge-stable --ozone-platform=x11 --inprivate
