--- org2	2026-04-05 19:01:45.595201324 +0200
+++ org.jellyfin.JellyfinServer.desktop	2026-04-05 19:03:48.387199626 +0200
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Name=Jellyfin Server
-Exec=/usr/bin/flatpak run --branch=stable --arch=x86_64 org.jellyfin.JellyfinServer
+Exec=/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=jellyfin org.jellyfin.JellyfinServer
 Type=Application
 Icon=org.jellyfin.JellyfinServer
 Categories=AudioVideo;
